graphqlapi/src/main/java/io/stargate/graphql/persistence/schemafirst/SchemaSourceDao.java

```diff
@@ -70,7 +70,10 @@ public SchemaSourceDao(DataStore dataStore) {
     return row.stream().map(r -> toSchemaSource(namespace, r)).collect(Collectors.toList());
   }
 
-  public SchemaSource getLatest(String namespace) throws Exception {
+  public SchemaSource getLatest(
```

Comment:
Since we provide the version this won't necessary be the latest. I think we can just call this method `get`.

Reply:
I've changed to `getByVersion`